public/static/locales/de/donate.json

```diff
@@ -55,5 +55,5 @@
   "plantTreesAtURL": "Pflanze Bäume unter {{url}}",
   "copiedToClipboard": "Text in die Zwischenablage kopiert!",
   "noProjectsFound": "Keine Projekte gefunden",
-  "inTimeOfTaxReturns": ""
+  "inTimeOfTaxReturns": "bekommen Sie rechtzeitig für ihre Steuererklärung"
```

Comment:
I made all the translations with "Du" and not "Sie" in the German language. We should keep that style and do not mix it!

Reply:
Besides we really should avoid fragments of sentences as some languages want to change the order of the words or the translators can not understand the context.

Reply:
Can you please make that change in Lingohub or let me know the string I'll update.

Reply:
Changed it in LingoHub.

Reply:
we go with Du. I've also asked Felix to review Lingohub